app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -6,7 +6,6 @@ import android.view.ViewGroup
 import androidx.appcompat.app.AppCompatActivity
```

Comment:
The merging of this file with develop has not been correct because of which there have been changes and as a result the tests are filing.
Suggest doing following things to fix this:
1. Copy content from `develop`: https://github.com/oppia/oppia-android/blob/develop/app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt
2. Paste in this your current branch `RecentlyPlayedFragmentPresenter` file.
3. Fix errors by changing the names.

Now run the earlier failing test and see if thats fixed or not? If yes, then make a push or else share screenshot of the failure.

Reply:
![Screenshot (44)](https://user-images.githubusercontent.com/56197993/111662464-f29dbc80-8835-11eb-854d-de6d261cdb06.png)
@rt4914  these tests are failing

Reply:
@priyanka0906 I am unable to see the changes as I have suggested. Can you make sure you push your code and after which I can check locally.

Reply:
Thanks the updated code works.